src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="/"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        />
```

Comment:
Keep your attributes correctly formatted. Since the `img` element has more than 2 attributes, each one should start on a new line with 2-space indentation relative to the tag.